src/main/java/com/gmail/val59000mc/scenarios/scenariolisteners/TimberListener.java

```diff
@@ -32,7 +37,12 @@ public void onBlockBreak(BlockBreakEvent e) {
     private int breakTree(Block block, int i) {
         int broken = 0;
         if (UniversalMaterial.isLog(block.getType())){
-            block.breakNaturally();
+            if (dropPlanks){
+                block.setType(Material.AIR);
+                block.getWorld().dropItem(block.getLocation(), new ItemStack(Material.OAK_PLANKS, 4));
```

Comment:
OAK_PLANKS doesn't exist in 1.8-1.12. So this change will cause issues on those versions.

Reply:
OK, I edited